godotproject/Player.gd

```diff
@@ -227,3 +241,8 @@ func _on_PlayerTongue_tongue_swing(global_tongue_position):
 	swing_pivot_position = global_tongue_position
 	start_swing()
 	pass # Replace with function body.
+
+
+func _on_InvulnerableTimer_timeout():
+	isVulnerable = true
```

Comment:
Make sure to touch base with @charasyn to make sure the timer implementation is consistent

Reply:
@charasyn, anything you want to talk about on this?

Reply:
I think it would be better to skip using the isVulnerable variable, and instead of 
```
if isVulnerable:
```
do 
```
if $InvulnerableTimer.is_stopped():
```